core/common/src/rpc/IModelReadRpcInterface.ts

```diff
@@ -71,18 +72,16 @@ export abstract class IModelReadRpcInterface extends RpcInterface {
   public async getToolTipMessage(_iModelToken: IModelRpcProps, _elementId: string): Promise<string[]> { return this.forward(arguments); }
   public async getViewThumbnail(_iModelToken: IModelRpcProps, _viewId: string): Promise<Uint8Array> { return this.forward(arguments); }
   public async getDefaultViewId(_iModelToken: IModelRpcProps): Promise<Id64String> { return this.forward(arguments); }
-  /** @internal */
```

Comment:
I just make request and cancel snap internal recently...we don't expect anyone to call these directly.

Reply:
The entire class is internal.